cpp/ql/integration-tests/header-variant-tests/clang-pch/test.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+def test(codeql, cpp):
+    os.mkdir("pch")
+    codeql.database.create(command=[
+        f'"{cpp.get_tool("extractor")}" --mimic-clang -emit-pch -o pch/a.pch a.c',
```

Comment:
minor cosmetic nit: here you could define once `extractor = f"{cpp.get_tool("extractor")} --mimic-clang"` and then reuse that for all commands.

This also avoids an extremely minor FS churn with calling `get_tool` repeatedly (that tries `tools/extractor` first with a couple of file extensions, and then goes into `tools/{CODEQL_PLAT}/extractor`, again potentially with multiple file extensions)

Reply:
Fixed in https://github.com/github/codeql/pull/19410/commits/edd18dc0521aa972fcee2a66b1aebf7fc44348a7. I left the duplication of `--mimic-clang`/`--mimic-cl`, as I feel that slightly better signals what is going on.